Examples/MuxPlayerSwiftExample/MuxPlayerSwiftExample.xcodeproj/project.pbxproj

```diff
@@ -551,7 +551,7 @@
 				ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES = YES;
 				CODE_SIGN_STYLE = Automatic;
 				CURRENT_PROJECT_VERSION = 1;
-				DEVELOPMENT_TEAM = XX95P4Y787;
```

Comment:
Were these added by mistake?

Reply:
For some reason I have a different developer team (name withheld)

I think both belong to Mux, and we should probably figure this out at some point when you get back